src/components/breadcrumb-item/breadcrumb-item.ts

```diff
@@ -62,7 +62,7 @@ export default class SlBreadcrumbItem extends LitElement {
                 class="breadcrumb-item__label breadcrumb-item__label--link"
                 href="${this.href}"
                 target="${this.target}"
-                rel=${ifDefined(this.target ? this.rel : undefined)}
+                rel=${ifDefined(typeof this.target !== 'undefined' ? this.rel : undefined)}
```

Comment:
One thing I'm worried about is if `this.target` is, for example, an element, it could be undefined or the result of something like `querySelector()`. In that case, explicitly checking for `undefined` won't catch a `null` value where the existing syntax would.

Sure, we could check for both but IMO the truthy syntax is easier to read and write, especially since the property is typed.

I'm torn on this one. It's probably better to relax this one to reduce the noise. 😕

Reply:
Just note that this rule also lead to most of the other strict boolean expressions all throughout the code. Do you have a similar concern with those too and should I revert them all? Or just some cases where you think it is too verbose in a ternary?

I'm trying to figure out which option I should do:

1. Revert every change that was done as a result of this rule and disable the rule.
2. Create a util function for something like `isTruthy` to use in the cases where we do truly want these truthy booleans and use it selectively.
3. Something else?

Reply:
I'm open to the utility approach if you think that will work better to reduce verbosity but leave more reasonable instances of it intact.

Reply:
Do you just want this reverted in ternaries that are embedded in the html? Or are there other places you'd like this change applied?